src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
```suggestion
  for (const action of actions) {
    switch (action.type) {
      case 'addProperties':
        // case code
        break;
```

You can reformat your conditions as switch...case statement. But it's up to you.

Reply:
Generally yes, but in this case I like the `if ` construct more, since there are only 3 options, but if there were more of them, I would use switch case =)